.github/workflows/ci-cassandra.yml

```diff
@@ -47,12 +47,15 @@ jobs:
 
     - name: Setup CODECOV_TOKEN
       uses: ./.github/actions/setup-codecov
-
-    - name: Upload coverage to codecov
-      uses: codecov/codecov-action@894ff025c7b54547a9a2a1e9f228beae737ad3c2 # v3.1.3
+    
+    - name: Retry action for codecov uploads
+      uses: Wandalen/wretry.action@v1.3.0
```

Comment:
We generally try to avoid 3rd-party actions.
* is this the only retry action available?
* if not, what other actions did you look at, how did you pick this one? Are there any official ones?

Reply:
hey @yurishkuro, I found this action to be good fit, if you have anything in mind please suggest one 🙂